mlxbf-bootctl.c

```diff
@@ -112,8 +112,11 @@ void die(const char* fmt, ...)
 #define LIFECYCLE_STATE_PATH "lifecycle_state"
 #define SECURE_BOOT_FUSE_STATE_PATH "secure_boot_fuse_state"
 
+#define INPUT_BFB_MAX_SIZE (20 * 1024 * 1024)
 #define MMC_BOOT_PARTITION_MAX_SIZE (4 * 1024 * 1024)
 
+#define DMI_PROCESSOR_INFO_TYPE 4
+
```

Comment:
These two lines seems not used?

Reply:
Forgot to remove it for this patch. This is WIP.